files/get_hiera/1/hieradata/common.yaml

```diff
@@ -1,2 +1,2 @@
 ---
-message: "Welcome to $::hostname. Don't break anything!"
+message: "Welcome to try.puppet.com. Don't break anything!"
```

Comment:
If you want this to be interpolated, you could use `"Welcome to %{::hostname}. Don't break anything!"`